src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString.split(';');
+  const newArr = sourceArr.map((el) => el.trim());
```

Comment:
[CODE KNOWLEDGE] - You don't need to make a copy before each operation. Moreover, methods like `map` don't mutate the original array, so you can chain the `map` directly after `split` without the need to assign it to a new variable.